### ISSUE TITLE
indices shift in sentences with spans

### ISSUE THREAD

**Issue by maryszmary** [jonorthwash/ud-annotatrix#63]
There are problems with graphical editing of sentences with supertokens, caused by the indices shift.
E.g, I change the deprel between *wan* and *na*
![image](https://user-images.githubusercontent.com/16348652/29477790-a77616be-8472-11e7-9d9e-cf5927b96f6c.png)

But another deprel is edited instead:
![image](https://user-images.githubusercontent.com/16348652/29477818-c3e08f0a-8472-11e7-9dcd-43917068cefc.png)

**Comment by maryszmary** [comment]
Fixed shifting issues for editing POS, token labels and deprels, drawing arcs and deleting arcs.

The only thing affected by the index shift issue now is merging and splitting tokens, which turned out to be a bit trickier. Happily, there're rarely more than 1 multiword token token per sentence, and for sentences without multiword token merging/splitting/merging into a multiword token works ok now.

**Comment by ftyers** [comment]
List of languages in UD by percentage of sentences with more than one multitoken per sentence (numbers approximate):

```
98.43	320	315	UD_Coptic
88.94	5725	5092	UD_Hebrew
77.86	6984	5438	UD_Arabic
45.99	8891	4089	UD_Portuguese
37.83	600	227	UD_Tamil
35.18	13402	4716	UD_Italian
19.35	31	6	UD_Kazakh
15.71	16031	2520	UD_French
15.09	2643	399	UD_French-Sequoia
10.91	15587	1701	UD_Spanish
5.088	14917	759	UD_German
4.442	4660	207	UD_Turkish
3.539	5397	191	UD_Persian
0.9821	7127	70	UD_Polish
0.7011	14832	104	UD_Catalan
0.1273	77765	99	UD_Czech
```

**Comment by maryszmary** [comment]
Oh. OK.
(Well, it's not that I wasn't going to fix it, it just was trickier).

**Comment by maryszmary** [comment]
Finally fixed in 173a5be.
